### DISCUSSION TITLE
Verticle support for osu maina?

### DISCUSSION THREAD

**Discussion by KrispyKebaeb** [ppy/osu#16787] [Q&A] [answered]
In the current release, vertical orientation is removed and landscape is locked, but this was the update to introduce mania for touch screen devices,  vertical support for mania would be pretty helpful, also why was vertical support removed? I enjoyed playing vertically sometimes.

**Comment by peppy** [comment, marked as answer]
Please see https://github.com/ppy/osu/discussions/16783. Still open to restoring portrait mode if there's support for it so add your opinion there.

**Reply by KrispyKebaeb** [reply]
Oh thank you, also i was wondering about osu lazer, the development is almost complete according to one of your public comments, i think the touch screen needs to be worked on now, todays update made me really happy, i hope yall continue to work on touch screen a bit more, it has been same for years, we wouldnt mind if touch screen rework will take some time, we have been waiting for years we can wait for more! Thank you and good luck.

**Reply by bdach** [reply]
desktop remains the priority as it has been "for years" now
